spec/apis/navigate-to-url.spec.js

```diff
@@ -0,0 +1,103 @@
+function hrefWithoutHash() {
```

Comment:
These tests were copied out of single-spa.spec.js. I also altered them to actually use the dom for anchor tags and click events so that we can do a more thorough test in all browsers.